recipes/libgeotiff/all/patches/fix-cmake-1.5.1.patch

```diff
@@ -79,24 +75,3 @@
        set_target_properties(
          ${GEOTIFF_LIBRARY_TARGET}
          PROPERTIES
-@@ -361,11 +363,15 @@ ENDIF(UNIX)
- SET_TARGET_PROPERTIES(${GEOTIFF_LIBRARY_TARGET} PROPERTIES
-    OUTPUT_NAME ${GEOTIFF_LIB_NAME})
- 
--TARGET_LINK_LIBRARIES(${GEOTIFF_LIBRARY_TARGET}
--    ${TIFF_LIBRARIES}
--    ${PROJ_LIBRARIES}
--    ${ZLIB_LIBRARIES}
--    ${JPEG_LIBRARIES})
-+find_package(TIFF REQUIRED)
-+target_link_libraries(${GEOTIFF_LIBRARY_TARGET} TIFF::TIFF)
-+find_package(proj4 QUIET CONFIG)
-+if(TARGET PROJ4::proj)
-+    target_link_libraries(${GEOTIFF_LIBRARY_TARGET} PROJ4::proj)
-+else()
-+    find_package(proj REQUIRED CONFIG)
-+    target_link_libraries(${GEOTIFF_LIBRARY_TARGET} PROJ::proj)
-+endif()
```

Comment:
You don't want to remove this. zlib & libjpeg are not direct dependencies of libgeotiff, it's old CMake style.

Reply:
@SpaceIm 
Thanks!
I fixed.